week-8/Homework/mandatory/1-practice/1-practice.md

```diff
@@ -26,6 +26,12 @@ The following endpoint is publicly available from Github
 
 <!-- Write your answer here -->
 
+404 status (Not found) returns
+
```

Comment:
Yes, if you put that URL in browser you will get a 404. This question was not clear enough, so I will mark you correct (because technically you are 😅).

This question should be worded as:
`Describe what kind of data you would expect this endpoint to return?`

So, looking at the way the endpoint is constructed, can you tell what kind of information the endpoint would return without calling the endpoint?